src/subscript/sector2fluxnum/flux_obj.py

```diff
@@ -0,0 +1,275 @@
+import sys
+import os
+
+from ecl.eclfile import EclKW
+from ecl.grid import EclRegion
+
+try:
```

Comment:
Can you add a comment about the rationale for this try/except? Is it a version dependency?